test/e2e/lib/creation.go

```diff
@@ -113,5 +112,5 @@ func (c *Client) CreateJobOrFail(job *batchv1.Job, options ...func(*batchv1.Job,
 	if _, err := jobs.Create(job); err != nil {
 		c.T.Fatalf("Failed to create job %q: %v", job.Name, err)
 	}
-	c.Tracker.Add("batch", "v1", "jobs", namespace, job.Name)
+	c.Tracker.Add("batch", "v1", "jobs", c.Namespace, job.Name)
```

Comment:
should be job.Namespace instead of c.Namespace?

Reply:
Fixed.